src/pages/questionPage.js

```diff
@@ -1,38 +1,72 @@
 'use strict';
 
 import {
-  ANSWERS_LIST_ID,
-  NEXT_QUESTION_BUTTON_ID,
-  USER_INTERFACE_ID,
+    ANSWERS_LIST_ID,
+    NEXT_QUESTION_BUTTON_ID,
+    USER_INTERFACE_ID,
 } from '../constants.js';
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
 
-export const initQuestionPage = () => {
-  const userInterface = document.getElementById(USER_INTERFACE_ID);
-  userInterface.innerHTML = '';
+export const initQuestionPage = (userInterface) => {
 
-  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+    userInterface = document.getElementById(USER_INTERFACE_ID);
+    userInterface.innerHTML = '';
 
-  const questionElement = createQuestionElement(currentQuestion.text);
+    const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
 
-  userInterface.appendChild(questionElement);
+    const questionElement = createQuestionElement(currentQuestion.text);
 
-  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+    userInterface.appendChild(questionElement);
+    let score = 0;
```

Comment:
This function gets called every time we move to the next question, so the score will keep getting reset to 0. We can fix this by making `score` a property of `quizData`, so that the variable has the same lifetime as the app.

Reply:
Done